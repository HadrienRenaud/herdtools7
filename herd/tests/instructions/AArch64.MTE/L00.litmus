AArch64 L00
Variant=mixed,mte,sync
{
int x[6]={1,2,3,4,5,6};
0:X1=x;
0:X2=x:red;
}
 P0              ;
 STZG  X2,[X1]   ;
 MOV W0,#7       ;
 STR W0,[X2]     ;
 STR W0,[X1,#20] ; 
forall x={7,0,0,0,5,7}

