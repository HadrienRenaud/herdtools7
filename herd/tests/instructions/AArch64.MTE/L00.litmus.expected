Test L00 Required
States 1
x={7,0,0,0,5,7};
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (x={7,0,0,0,5,7})
Observation L00 Always 1 0
Hash=686ab9d457f82594fb739b04bbe5a40b

